### PR TITLE
chore(agents): add code review skills

### PR DESCRIPTION
**Pull request by bjoerge**
### Description
Adds code review skills from https://github.com/addyosmani/agent-skills#review---quality-gates-before-merge

I've had good experience with the code-review-and-quality skill previously, not tried the other ones.

Skipping the [Security and hardening](https://github.com/addyosmani/agent-skills/blob/main/skills/security-and-hardening/SKILL.md) skill since its more geared towards backend services

### What to review
Do we agree with the value judgements of these? I.e. the Code simplification skill calls out usage of nested ternaries, which we recently removed an eslint rule for.

I'd say lets try, and remove/tweak if they don't work out for us.

### Testing
n/a

### Notes for release
n/a

### CODE REVIEW

File: .agents/skills/code-review-and-quality/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+## Honesty in Review
+
+When reviewing code — whether written by you, another agent, or a human:
+
+- **Don't rubber-stamp.** "LGTM" without evidence of review helps no one.
+- **Don't soften real issues.** "This might be a minor concern" when it's a bug that will hit production is dishonest.
+- **Quantify problems when possible.** "This N+1 query will add ~50ms per item in the list" is better than "this could be slow."
+- **Push back on approaches with clear problems.** Sycophancy is a failure mode in reviews. If the implementation has issues, say so directly and propose alternatives.
+- **Accept override gracefully.** If the author has full context and disagrees, defer to their judgment. Comment on code, not people — reframe personal critiques to focus on the code itself.
+
```
Comment (RitaDias) [inline .agents/skills/code-review-and-quality/SKILL.md:255]: I've never written skills before so might be over indexing but I also found that as a last thing that it should do in review regaridng honesty is: are the tests that are here enough? Are there missing use cases that should be tested now that were missed in an initial pass? 
Not necessarily a "we should have had this test to begin with before our changes" but a "you have introduced code and this use case feels like something that could be an edge case or even a common case that you just have no considered)

**Review by RitaDias — COMMENTED** [review]
I love this, I want it, I left a comment but again I want this 👐

File: .agents/skills/code-review-and-quality/SKILL.md (hunk 1 above)
Comment (bjoerge) [reply .agents/skills/code-review-and-quality/SKILL.md:255]: Good point. Lets see how well it already works in this regard?

**Review by RitaDias — APPROVED** [review]
